sdk/eventhub/event-hubs/src/eventHubReceiver.ts

```diff
@@ -250,175 +246,63 @@ export class EventHubReceiver extends LinkEntity {
 
   private _onAmqpError(context: EventContext): void {
     const rheaReceiver = this._receiver || context.receiver;
-    if (!rheaReceiver) {
-      return;
-    }
-
-    const amqpError = rheaReceiver.error;
-    if (!amqpError) {
-      return;
-    }
-
-    if (rheaReceiver.isItselfClosed()) {
```

Comment:
Just to make sure I understand, is this check no longer needed because calling close on the receiver will clear the handlers?

Reply:
yes